src/organization-domains/interfaces/organization-domain.interface.ts

```diff
@@ -0,0 +1,22 @@
+export enum VerificationState {
```

Comment:
Small suggestion here to make this naming more consistent with our other entities.
```suggestion
export enum OrganizationDomainState {
```